README.md

```diff
@@ -80,7 +80,7 @@ Run the agent in local mode to audit a GitHub repository directly.
 You can use the following example repository to test out the agent. The results will be saved in JSON format in the specified output file, by default that is `audit.json`.
 
 ```bash
-audit-agent local --repo https://github.com/andreitoma8/learn-solidity-hacks.git --output audit.json
```

Comment:
I wouldn't replace this, it's a useful reference to a set of common Solidity vulnerabilities compiled by one of our colleagues. People might find it useful, especially if they are really new to smart contract auditing.